pyanalyze/__main__.py

```diff
@@ -1,7 +1,12 @@
 import sys
+from os import system
 
 from pyanalyze.name_check_visitor import NameCheckVisitor
 
+# Enable ANSI color codes for Windows cmd using this strange workaround
+# ( see https://github.com/python/cpython/issues/74261 )
+system("")
```

Comment:
Let's do this only on Windows to avoid doing unnecessary work on other OSes, and let's do it inside the main() function so this doesn't get run if the file is merely imported. That's important for pyanalyze because it type checks by importing modules.

Also, I no longer work for Quora and won't be able to merge it, but after I approve I can prod someone to get it merged.